zomes/wiki/code/src/user.rs

```diff
@@ -82,7 +81,7 @@ pub fn create_user_if_non_existent(data: String) -> ZomeApiResult<Address> {
     let address = User(data.clone()).entry().address();
     match hdk::get_entry(&address)? {
         None => {
-            let page_anchor = utils::anchor("users", "all_users")?;
+            let page_anchor = holochain_anchors::anchor("users".into(), "all_users".into())?;
```

Comment:
We don't need to create a new `User` entry, we only need to create usernames as anchors using the holochain_anchors library.

Reply:
How do we link to the agent's address?

Reply:
Uhh that's right